GOVERNANCE.md

```diff
@@ -0,0 +1,147 @@
+# Governance
+
+## Overview
+
+BlockNote is an open-source project stewarded by the founding team with the goal of growing into a contributor-led ecosystem over time. Governance is designed to balance long-term technical integrity, transparency, and sustainability while remaining lightweight at the project’s current scale.
```

Comment:
I'm not sure "growing into a contributor-led ecosystem" is an explicit goal at this moment (or we should further clarify it). Point in case; if there's a profitable organization (like OpenBlocks) supporting it, is it contributor led?